### DISCUSSION TITLE
writing/2024/10/25/running-effective-ai-standups/

### DISCUSSION THREAD

**Discussion by giscus** [jxnl/blog#87] [General]
# writing/2024/10/25/running-effective-ai-standups/

Learn how to enhance AI engineering processes through rigorous and insightful updates, focusing on hypotheses, interventions, results, and trade-offs.

https://jxnl.co/writing/2024/10/25/running-effective-ai-standups/

**Comment by Abdulk084** [comment]
This speaks my heart out. thanks for verbalising the pain of AI Applied scientists working with dev/engineering teams.
